bitnami/thanos/values.yaml

```
@@ -1834,6 +1847,18 @@ storegateway:
     ## @param storegateway.pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
     ##
     maxUnavailable: ""
+  ## Sharded parameters
+  ## @param storegateway.sharded.enabled Enable sharding for Thanos Store Gateway
+  ## @param storegateway.sharded.hashPartioning.shards Setting hashPartioning will create multiple store deployments based on the number of shards specified using the hashmod of the blocks
+  ## @param storegateway.sharded.timePartioning [array] Setting time timePartioning will create multipale store deployments based on the number of partions
```

Fix a couple of typos:
```suggestion
  ## @param storegateway.sharded.hashPartioning.shards Setting hashPartioning will create multiple store statefulsets based on the number of shards specified using the hashmod of the blocks
  ## @param storegateway.sharded.timePartioning [array] Setting time timePartioning will create multiple store deployments based on the number of partitions
```